src/components/Sidebar/NavContent.jsx

```diff
@@ -59,44 +47,6 @@ function NavContent({ handleDrawerToggle }) {
     }, 60000);
     return () => clearTimeout(interval);
   });
-  const checkPage = useCallback((match, location, page) => {
-    const currentPath = location.pathname.replace("/", "");
-    if (currentPath.indexOf("dashboard") >= 0 && page === "dashboard") {
-      return true;
-    }
-    if (currentPath.indexOf("zap") >= 0 && page === "zap") {
-      return true;
-    }
-    if (currentPath.indexOf("stake") >= 0 && page === "stake") {
-      return true;
-    }
-    if (currentPath.indexOf("wrap") >= 0 && page === "wrap") {
-      return true;
-    }
-    if (currentPath.indexOf("give") >= 0 && page == "give") {
-      return true;
-    }
-    if (currentPath.indexOf("givedonations") >= 0 && page == "give/donations") {
-      return true;
-    }
-    if (currentPath.indexOf("giveredeem") >= 0 && page == "give/redeem") {
-      return true;
-    }
-    if ((currentPath.indexOf("bonds-v1") >= 0 || currentPath.indexOf("choose_bond") >= 0) && page === "bonds-v1") {
-      return true;
-    }
-    if ((currentPath.indexOf("bonds") >= 0 || currentPath.indexOf("choose_bond") >= 0) && page === "bonds") {
-      return true;
-    }
-    if (currentPath.indexOf("33-together") >= 0 && page === "33-together") {
-      return true;
-    }
-    if (currentPath.indexOf("33-together") >= 0 && page === "33-together") {
-      return true;
-    }
-    return false;
-  }, []);
-
```

Comment:
I couldn't tell what this block was doing and removing it had no effect for active styles, etc. Our NavItem component relies on react-router-dom under the hood (so it'll apply active css classes for us)

Reply:
ah thanks